vpr/SRC/route/route_common.c

```diff
@@ -439,7 +420,24 @@ void pathfinder_update_one_cost(struct s_trace *route_segment_start,
 
 	} /* End while loop -- did an entire traceback. */
 }
+void pathfinder_update_single_node_cost(int inode, int add_or_sub, float pres_fac) {

```

Comment:
Thanks for refactoring!  Good to split out the routine, but I agree with Oleg and Kevin's comments.